client/coral-settings/containers/SettingsContainer.js

```diff
@@ -0,0 +1,63 @@
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+
+import {} from '../../coral-framework/actions/embedStream';
```

Comment:
This will import the actions that embedStream will have in the future. As I explained, this is a PR without the functionality for bio. So there is where we should grab the actions for CommentHistory and Bio